src/database.rs

```diff
@@ -0,0 +1,10 @@
+use sqlx::{Error, postgres::PgPool};
+
+use crate::constants;
+
+
+pub async fn get_conn_pool() -> Result<PgPool, Error> {
+    let pool = PgPool::connect(constants::DB_URL).await?;
```

Comment:
Please add an option to specify PostgreSQL username and password via env
https://docs.rs/sqlx/latest/sqlx/postgres/struct.PgConnectOptions.html#method.username

Reply:
According to [Docker PostgreSQL Readme](https://github.com/docker-library/docs/blob/master/postgres/README.md#postgres_password) password can be omitted if client is on the same host. I leave it up to you if you want to override default username or not

Reply:
It now checks the default Postgres environment variables for values and fallbacks to the hardcoded url